libs/llrt_numbers/src/lib.rs

```diff
@@ -295,6 +300,9 @@ mod test {
         }
```

Comment:
Interestingly, I think the test on the line above would have failed if `0` was ever chosen, but I guess it hasn't been one of the million random numbers yet.

Reply:
Maybe why we shouldn't always just test with random data :)